vyxal/helpers.py

```diff
@@ -8,7 +8,6 @@
 import collections
 import inspect
 import itertools
-import math
```

Comment:
why?

Reply:
Lgtm said it was an unnecessary import, and a quick ctrl-f in there found nothing but description comments.